### DISCUSSION TITLE
弯道超车，直接到博主的最后一步

### DISCUSSION THREAD

**Discussion by frankelinli** [mebtte/animal-photosynthesis#26] [General]
我不是程序员，对编程都是自学。于是乎博主说的博客架构变化，一知半解。但是我直接跳到了最后一步，使用开源的静态博客生成器。跟博主殊途同归。。。。

**Comment by mebtte** [comment]
实践下来, 确实是静态页面生成器才是最符合博客的架构

**Comment by frankelinli** [comment]
运行一段时间 ，发现静态页面也够烦人的。发布内容和更改内容，是个大麻烦，即使更改一个字，也要重新build整个站点。

**Reply by mebtte** [reply]
文章数量少的情况下, 完整构建所有博文也是可以接受的
如果数量太多导致构建缓慢, 可以考虑一下缓存, 检测到文章变化才进行构建
